docs/docs/glossary.mdx

```diff
@@ -45,13 +65,11 @@ description: Glossary for all Rasa-related terms
 
 ## [Entity](./domain.mdx#entities)
 
-  Structured information that can be extracted from a user message.
-
-  For example a telephone number, a person's name, a location, the name of a product
+  Keywords that can be extracted from a user message. For example: a telephone number, a person's name, a location, the name of a product
```

Comment:
Information seemed like it could be interpreted to mean a lot of things, when really we're trying to say that we want to pluck out specific terms or words from the surrounding text. Is there a reason why we avoid "keywords" when talking about entities - is it not quite accurate?

Reply:
it sounds kind of weird to me, but if you think it's fine then let's go with it